src/hive/types.js

```diff
@@ -120,6 +125,24 @@ class BeehiveDirective extends SchemaDirectiveVisitor {
 
 }
 
+class BeehiveDeleteDirective extends SchemaDirectiveVisitor{
+
+  visitFieldDefinition(field, details) {
+    const target_type_name = this.args.target_type_name
+    const schema = this.schema
+    const table_config = this.schema._beehive.tables[target_type_name]
+
+    field.resolve = async function (obj, args, context, info) {
+            const table_config = schema._beehive.tables[target_type_name]
+            delete schema._beehive.tables[target_type_name]
```

Comment:
this would delete the table config not the object in the DB.  This should call a deleteItem method in pgsql, which will need to be created.